docs/components/Dialog.md

```diff
@@ -45,9 +45,8 @@ export default class MyApp extends React.Component {
         confirmIcon="check"
         confirmText="Yes"
         title="Please confirm:"
-        text="Dismiss the notification ?"
```

Comment:
`text` should be renamed to `message` + no extra space before "?" :)
`dialogType` should be renamed to `type`
`dialogLayout` should be renamed to `layout`

Reply:
Please remember to update Dialog docs (text -> message).

Reply:
Done !